### PR TITLE
Allow having conditional compilation and other method level attributes

### PR DESCRIPTION
**Pull request by nearmax**

### CODE REVIEW

File: near-bindgen-core/src/initializer_attribute.rs [hunk 1]
```diff
@@ -54,6 +54,13 @@ pub fn process_init_method(
     impl_type: &Type,
     is_trait_impl: bool,
 ) -> syn::Result<TokenStream2> {
+    let attrs = method.attrs.iter().fold(TokenStream2::new(), |mut acc, attr| {
+        let attr_str = attr.path.to_token_stream().to_string();
+        if attr_str != "callback_args_vec".to_string() && attr_str != "callback_args".to_string() {
```
Comment (frol) [inline near-bindgen-core/src/initializer_attribute.rs]: You don't need `&str.to_string()` to compare. `if attr_str != "callback_args_vec" && attr_str != "callback_args" {` should be just fine.

Comment (nearmax) [reply near-bindgen-core/src/initializer_attribute.rs]: Fixed.

File: near-bindgen-core/src/initializer_attribute.rs [hunk 2]
```diff
@@ -54,6 +54,13 @@ pub fn process_init_method(
     impl_type: &Type,
     is_trait_impl: bool,
 ) -> syn::Result<TokenStream2> {
+    let attrs = method.attrs.iter().fold(TokenStream2::new(), |mut acc, attr| {
+        let attr_str = attr.path.to_token_stream().to_string();
+        if &attr_str != "callback_args_vec" && &attr_str != "callback_args" {
```
Comment (frol) [inline near-bindgen-core/src/initializer_attribute.rs]: Hmm, why `&attr_str`?

Comment (nearmax) [reply near-bindgen-core/src/initializer_attribute.rs]: Fixed. That's weird it works, but CLion complains: https://www.dropbox.com/s/m1yo93wbtrnhf3o/Screenshot%202019-11-18%2016.15.19.png?dl=0

**Pull request merged**